core/aggregator/api/api_product.go

```diff
@@ -24,8 +24,29 @@ import (
 
 func CreateProduct(w http.ResponseWriter, r *http.Request) {
```

Comment:
don't forget to get the git configuration from parent workspace on product creation.